build.gradle

```diff
@@ -4,9 +4,9 @@ allprojects {
 }
 
 buildscript {
-    ext.kotlin_version = '1.2.50'
-    ext.dokka_version = '0.9.16'
-    ext.detekt_version = '1.0.0.RC5'
+    ext.kotlin_version = '1.2.61'
```

Comment:
@evant I'm happy to revert `kotlin` version, but like i said. I think an argument can be made that upgrading to slowly is also a risk. I think it's ok to be conservative when upgrading to a major version like the upcoming `1.3.x` release, as there could be breaking changes. The `kotlin` team (in my opinion) is pretty conservative with releases already and many of the updates actually fix critical bugs. Let me know what you decision is, i'm ok with reverting this if you feel strongly about it